content/open-source/oci/what-are-oci-artifacts.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "What are OCI Artifacts?"
+type: "article"
+description: "OCI Artifacts are a way of using OCI registries, or container registries that are compliant with specifications set by the Open Container Initiative, to store arbitrary files."
+date: 2022-06-09T15:22:20+01:00
+lastmod: 2022-06-09T15:22:20+01:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "oci"
+weight: 150
+toc: true
+---
+
+[OCI artifacts](http://github.com/opencontainers/artifacts) are a way of using OCI registries, or container registries that are compliant with specifications set by the [Open Container Initiative](https://edu.chainguard.dev/open-source/oci/what-is-the-oci/), to store arbitrary files. While container registries were originally designed to store and distribute images, container engineers are increasingly using them to store non-container files such as Helm charts, Tekton bundles, and policy modules. By storing associated objects in the same infrastructure as containers, container engineers can consolidate their security and management efforts. Another benefit of using OCI registries is that provide a [content-addressable API](https://edu.chainguard.dev/open-source/oci/what-is-the-oci/#image-manifest), a way of referring to files that assures their authenticity and integrity.  
+
+OCI artifacts are sometimes misunderstood as a new specification or format but they are in fact just a way of using the OCI Image Specification to store something other than an image in a container registry. The Image Specification requires a particular format for the [Image Manifest](https://edu.chainguard.dev/open-source/oci/what-is-the-oci/#image-manifest), and the inclusion of the specified OCI `mediaTypes` values `application/vnd.oci.image.config.v1+json` and `application/vnd.oci.image.layer.v1.tar+gzip` in the `config` and `layers` fields. To store an artifact using the Image Specification, you need to change the `mediaTypes` values in the `config` and `layers` fields as in the example manifest below.  
```

Comment:
Not sure if we need "just" for emphasis, but up to you. Do you want to spell out that the `mediaTypes` values are for JSON data and compressed files?

```suggestion
OCI artifacts are sometimes misunderstood as a new specification or format but they are in fact a way of using the OCI Image Specification to store something other than an image in a container registry. The Image Specification requires a particular format for the [Image Manifest](https://edu.chainguard.dev/open-source/oci/what-is-the-oci/#image-manifest), and the inclusion of the specified OCI `mediaTypes` values `application/vnd.oci.image.config.v1+json` and `application/vnd.oci.image.layer.v1.tar+gzip` in the `config` and `layers` fields. To store an artifact using the Image Specification, you need to change the `mediaTypes` values in the `config` and `layers` fields as in the example manifest below.  
```